index.php

```diff
@@ -1,379 +1,28 @@
 <html>
+<?php
+	header("Link:" .
+	"<https://fonts.gstatic.com>; rel=preconnect," .
+	"<img/gurka.png>; rel=prefetch," .
+	"<img/moon.png>; rel=prefetch," .
+	"<img/sun.png>; rel=prefetch"
+	);
+?>
```

Comment:
Hela denna PHP del ger ingen funktionalitet alls, endast gör att bilderna laddas in snabbare.
Detta går alltså att ta bort vilket då också gör det möjligt att läsa filen som `.html`